sormas-backend/src/main/java/de/symeda/sormas/backend/systemevent/SystemEventFacadeEjb.java

```diff
@@ -121,7 +122,6 @@ public SystemEvent fromDto(@NotNull SystemEventDto source, SystemEvent target, b
 	}
 
 	public SystemEventDto toDto(SystemEvent source) {
-
```

Comment:
This formatting violates [formatting guidelines](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md#development-contributing-guidelines)
> When defining a method, enter a blank line before starting to write its body (except for methods with only one line of code, e.g. most getters and setters).